x-pack/plugins/security_solution/public/resolver/store/data/selectors.ts

```diff
@@ -672,15 +675,18 @@ export const nodesAndEdgelines: (
 /**
  * If there is a pending request that's for a entity ID that doesn't matche the `entityID`, then we should cancel it.
  */
-export function databaseDocumentIDToAbort(state: DataState): string | null {
+export function treeRequestParametersToAbort(state: DataState): TreeFetcherParameters | null {
```

Comment:
Similarly, the logic to determine if we need to abort a pending request considers the `databaseDocumentID` **and** the `indices`.